index.html

```diff
@@ -0,0 +1,31 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <title>Template 3</title>
+</head>
+<body>
+    <header class="topbar">
+        <h3>Welcome</h3>
+        <nav class="menu"><img src="./images/icons/Union.png" alt="menu"></nav>
+    </header>
+    <ul class="main">
+        <li class="st">
+            <h1 class="t1">Hey there. I’m Sameh</h1>
+            <h1 class="t2">I’m a software developer</h1></li>
```

Comment:
- [ ] While this works, it's considered good practice to have only one `h1` tag in your page. You can use an `h2` tag here instead of the second `h1` tag.